This file is part of MXE.
See index.html for further information.

From 9e74dc0c9cda72506303a276a0a6174b500fae15 Mon Sep 17 00:00:00 2001
From: Boris Nagaev <bnagaev@gmail.com>
Date: Sun, 30 Aug 2015 01:59:15 +0200
Subject: [PATCH 2/3] remove unused members of peerlistwidget

They were still generated by MOC and caused compilation errors
---
 src/gui/properties/peerlistwidget.h |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/gui/properties/peerlistwidget.h b/src/gui/properties/peerlistwidget.h
index 0771555..7f5f771 100644
--- a/src/gui/properties/peerlistwidget.h
+++ b/src/gui/properties/peerlistwidget.h
@@ -83,11 +83,6 @@ protected slots:
   void saveSettings() const;
   void showPeerListMenu(const QPoint&);
 
-#if LIBTORRENT_VERSION_NUM < 10000
-  void limitUpRateSelectedPeers(const QStringList& peer_ips);
-  void limitDlRateSelectedPeers(const QStringList& peer_ips);
-#endif
-
   void banSelectedPeers(const QStringList& peer_ips);
   void handleSortColumnChanged(int col);
 
-- 
1.7.10.4

